### ISSUE TITLE
Namespace changed with change in location of the schema

### ISSUE THREAD

**Issue by denevers** [opengeospatial/boreholeie#61]
the namespace of the new schema in GitHub (https://github.com/opengeospatial/boreholeie/blob/master/schemas/BhML-Core.xsd ) is now

https://raw.githubusercontent.com/opengeospatial/boreholeie/master/schemas

Is this the namespace we should use for examples (or should we care ?)

**Comment by sgrellet** [comment]
Yes, that's what we decided during the last webconf. To ensure that everything pertaining to the material produced during the ER persists in a common space.

**Comment by bsimons14** [comment]
I'm getting a validation error {element html} from:
xsi:schemaLocation="https://raw.githubusercontent.com/opengeospatial/boreholeie/master/schemas https://github.com/opengeospatial/boreholeie/blob/master/schemas/BhML-Core.xsd

I assume because it is pointing to the github page rather than the actual schema file?

**Comment by bsimons14** [comment]
OOps Should have used:
https://raw.githubusercontent.com/opengeospatial/boreholeie/master/schemas/BhML-Core.xsd

**Comment by denevers** [comment]
> 
> 
> Yes, that's what we decided during the last webconf. To ensure that everything pertaining to the material produced during the ER persists in a common space.

yes, for the schema location, but it did not realise it imply changing the namespace as well.  But it's no big deal, it can be done at the same time schemaLocation is updated
